google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -431,6 +487,32 @@ static QueryParameterValue fromPb(
         }
         valueBuilder.setArrayValues(arrayValues.build());
       }
+    } else if (type == StandardSQLTypeName.STRUCT) {
+      Map<String, QueryParameterValue> structValues = new HashMap<>();
```

Comment:
This map is used only inside else block from line#503, it would be better to declare it inside that block only.

Reply:
Done